lib/chef/knife/ec2_server_create.rb

```diff
@@ -67,10 +67,18 @@ class Ec2ServerCreate < Knife
         :proc => Proc.new { |groups| groups.split(',') }
 
       option :security_group_ids,
-        :short => "-g 'X,Y,Z'",
-        :long => "--security-group-ids 'X,Y,Z'",
-        :description => "The security group ids for this server; required when using VPC,Please provide values in format --security-group-ids 'X,Y,Z'",
-        :proc => Proc.new { |security_group_ids| security_group_ids.split(',') }
+        :short => "-g SECURITY_GROUP_IDS",
+        :long => "--security-group-ids",
```

Comment:
@btm 
1) So should I show message as deprecated for option --security-group-ids for all kinds of input, whether its comma separated or single value ?
2) In knife.rb the option will remain the same as security_group_ids and it will accept array.
3) Will add the Deprecated message in the --security-group-ids description "[DEPRECATED] Use --security-group-id option instead"